admin/js/flizpay-admin.js

```diff
@@ -61,113 +48,129 @@
-    apiKeyInput.parentNode.appendChild(testButton);
-    apiKeyInput.parentNode.appendChild(resultField);
-    webhookURLInput.setAttribute("disabled", true);
-    webhookURLInput.setAttribute("type", "hidden");
-    enabledCheckbox.setAttribute("disabled", true);
-    webhookAlive.setAttribute("disabled", true);
     const divider = document.createElement("hr");
     const divider2 = document.createElement("hr");
-    divider.setAttribute("style", "width: 80vw");
-    divider2.setAttribute("style", "width: 80vw");
     const dividerRow = document.createElement("tr");
     const dividerRow2 = document.createElement("tr");
-    dividerRow.append(divider);
-    dividerRow.append(exampleImage);
-    dividerRow2.append(divider2);
-    document
-      .querySelector("table > tbody > tr:nth-child(3)")
-      .insertAdjacentElement("afterend", dividerRow);
-    document
-      .querySelector("table > tbody > tr:nth-child(7)")
-      .insertAdjacentElement("afterend", dividerRow2);
+    const exampleImage = document.createElement("img");
 
-    if (webhookAlive.getAttribute("checked")) {
-      description.setAttribute(
-        "style",
-        "color: #001F3F; background-color: #80ED99; padding: 10px; font-weight: bold; margin-top: 30px;"
-      );
-      description.innerHTML = descriptionText;
+    initCustomAttributesAndStyles();
+
+    if (isConnectionFailed()) {
+      renderConnectionFailed();
+    } else if (isConnectionPending() && reachedMaxAttempts()) {
+      renderConnectionFailed();
+    } else if (isConnectionPending()) {
+      renderWaitingConnection();
+      scheduleReloadAndIncreaseCounter();
+    } else {
+      localStorage.removeItem("flizpay_admin_connection_attempts");
     }
-    displayHeadlineLabel.setAttribute(
-      "style",
-      displayHeadlineInput.checked ? "display: none;" : "display: block;"
-    );
 
-    jQuery(displayHeadlineInput).on("change", () => {
-      if (!displayHeadlineInput.checked) {
-        displayHeadlineLabel.setAttribute("style", "display: block;");
-      } else {
-        displayHeadlineLabel.setAttribute("style", "display: none;");
+    $("form").on("submit", (e) => {
+      if (hasChangedApiKey() && !confirm(confirmReconfigurationText)) {
+        e.preventDefault();
       }
     });
 
-    $(".woocommerce-save-button").on("click", function (e) {
-      e.preventDefault();
-      e.stopImmediatePropagation();
+    function reachedMaxAttempts() {
+      return connectionAttempts === 10;
+    }
+
+    function hasChangedApiKey() {
+      return (
+        webhookAlive.checked &&
+        document.querySelector("#woocommerce_flizpay_flizpay_api_key").value !==
+          initialApiKeyValue
+      );
+    }
+
+    function renderConnectionFailed() {
+      localStorage.removeItem("flizpay_admin_connection_attempts");
+      resultField.classList.add("connection-failed");
+      resultField.innerHTML = `
+        ${failedConnectionText}
+        <image src='${flizpayParams.loading_icon}' />
```

Comment:
Shouldnt this tag be img?

Reply:
Most browsers will parse it to img. But I fixed